mParticle-Google-Analytics-Firebase-GA4/MPKitFirebaseGA4Analytics.m

```diff
@@ -358,6 +371,54 @@ - (void)logUserAttributes:(NSDictionary<NSString *, id> *)userAttributes {
     }
 }
 
+- (MPKitExecStatus *)setConsentState:(nullable MPConsentState *)state {
+    [self updateConsent];
+    
+    return [self execStatus:MPKitReturnCodeSuccess];
+}
+
+- (void)updateConsent {
+    // If Defaults are not set do not send FIRAnalytics Consent
+    if (!self.configuration[kMPFIRGA4DefaultAdStorageKey] ) {
```

Comment:
I added this to help clients who update but don't realize Google has a new consent feature. Basically, If the defaults aren't set we don't ever send consent to Google. Otherwise we'd always be sending "Denied" which could affect their data.

What are people's thoughts on this?